### DISCUSSION TITLE
Set preferred Search option

### DISCUSSION THREAD

**Discussion by marcelweikum** [dailydotdev/daily#1513] [Feature Requests]
I would like to have an option to set the preferred search option in the browser extension.

The standard is, when I press enter it automatically searches for posts, but I want daily.dev as standard page for new tabs and want the search bar to automatically redirect to Google when i type and press enter, without actively selecting one of the options.

Standard:
![{BEA23714-3FA2-45C5-867E-ABB184418DE4}](https://github.com/user-attachments/assets/36994eed-2afa-4f1e-af21-827efc93c8a6)

When explicitly selecting Google (this should be a feature to change in settings to automatically select):
![{BC6120FC-7CA7-41FB-BB52-C2FC07AA1F46}](https://github.com/user-attachments/assets/518c0be6-c2a6-4171-b4c1-8ede2c49b8ec)

**Comment by idoshamun** [comment]
Hi @marcelweikum,

We will consider it. Thanks for the suggestion :)
